bench/Alice/AliceBenchTask.cs

```diff
@@ -0,0 +1,96 @@
+﻿using System.Collections;
+using System.IO;
+using System.Text;
+using System.Text.RegularExpressions;
+using BenchmarkDotNet.Attributes;
+
+namespace Alice
+{
+    [CoreJob, ClrJob]
+    [MemoryDiagnoser]
+    [RPlotExporter, RankColumn, AsciiDocExporter, MarkdownExporter]
+    public class AliceBenchTask
+    {
+        private static string AliceFilename = "Alice's_Adventures_in_Wonderland.txt";
+        private static string text;
+
+        private Regex dotnetRegex;
+        private Regex dotnetRegexNoCompile;
+        private IronRure.Regex rustRegex;
+        private IronRure.Regex rustRegexUnicode;
+
+        [GlobalSetup]
+        public void GlobalSetup()
+        {
+            text = File.ReadAllText(AliceFilename, Encoding.UTF8);
+
+            InitializeRegex();
+        }
+
+        [Params(
+            @"\b(\w+)-legged",
+            //@"Alice",
```

Comment:
Can you uncomment all of these expressions.

Reply:
Ummm... it took me took long to generate a report, the report in this PR matches the code, if you have time can u uncomment the code and generate the report again?

Reply:
i will try and run these tomorrow. would be nice if it was possible to run more than one test per subprocess. I think it is the recompilation which takes time.